server/src/main/java/org/elasticsearch/cluster/routing/IndexShardRoutingTable.java

```diff
@@ -200,7 +204,16 @@ public int getSize() {
     }
 
     /**
-     * Returns a {@link List} of assigned shards
+     * Returns a {@link List} of active shards
+     *
+     * @return a {@link List} of shards
+     */
+    public List<ShardRouting> getActiveShards() {
```

Comment:
is this used somewhere? or what is the reason to add it?

Reply:
You are right it is not used, I accidentlly included with a cherry-pick